3scaleAdapter/pkg/threescale/threescale.go

```diff
@@ -37,9 +37,10 @@ type (
 
 	// Threescale contains the Listener and the server
 	Threescale struct {
-		listener net.Listener
-		server   *grpc.Server
-		client   *http.Client
+		listener   net.Listener
+		server     *grpc.Server
+		client     *http.Client
+		proxyCache *proxyConfigController
```

Comment:
can we have this value set to nil (ie. no cache) and everything else working just fine?

Reply:
Thats a good point I can do that